src/System.ObjectModel/ref/System.ObjectModel.cs

```diff
@@ -143,36 +143,36 @@ public enum NotifyCollectionChangedAction
-        public NotifyCollectionChangedEventArgs(System.Collections.Specialized.NotifyCollectionChangedAction action, object newItem, object oldItem, int index) { }
+        public NotifyCollectionChangedEventArgs(System.Collections.Specialized.NotifyCollectionChangedAction action, System.Collections.IList? changedItems, int startingIndex) { }
+        public NotifyCollectionChangedEventArgs(System.Collections.Specialized.NotifyCollectionChangedAction action, System.Collections.IList? changedItems, int index, int oldIndex) { }
+        public NotifyCollectionChangedEventArgs(System.Collections.Specialized.NotifyCollectionChangedAction action, object? changedItem) { }
+        public NotifyCollectionChangedEventArgs(System.Collections.Specialized.NotifyCollectionChangedAction action, object? changedItem, int index) { }
+        public NotifyCollectionChangedEventArgs(System.Collections.Specialized.NotifyCollectionChangedAction action, object? changedItem, int index, int oldIndex) { }
+        public NotifyCollectionChangedEventArgs(System.Collections.Specialized.NotifyCollectionChangedAction action, object? newItem, object? oldItem) { }
+        public NotifyCollectionChangedEventArgs(System.Collections.Specialized.NotifyCollectionChangedAction action, object? newItem, object? oldItem, int index) { }
         public System.Collections.Specialized.NotifyCollectionChangedAction Action { get { throw null; } }
-        public System.Collections.IList NewItems { get { throw null; } }
+        public System.Collections.IList? NewItems { get { throw null; } }
         public int NewStartingIndex { get { throw null; } }
-        public System.Collections.IList OldItems { get { throw null; } }
+        public System.Collections.IList? OldItems { get { throw null; } }
         public int OldStartingIndex { get { throw null; } }
     }
-    public delegate void NotifyCollectionChangedEventHandler(object sender, System.Collections.Specialized.NotifyCollectionChangedEventArgs e);
+    public delegate void NotifyCollectionChangedEventHandler(object? sender, System.Collections.Specialized.NotifyCollectionChangedEventArgs e);
 }
 namespace System.ComponentModel
 {
     public partial class DataErrorsChangedEventArgs : System.EventArgs
     {
-        public DataErrorsChangedEventArgs(string propertyName) { }
-        public virtual string PropertyName { get { throw null; } }
+        public DataErrorsChangedEventArgs(string? propertyName) { }
+        public virtual string? PropertyName { get { throw null; } }
     }
     public partial interface INotifyDataErrorInfo
     {
         bool HasErrors { get; }
-        event System.EventHandler<System.ComponentModel.DataErrorsChangedEventArgs> ErrorsChanged;
-        System.Collections.IEnumerable GetErrors(string propertyName);
+        event System.EventHandler<System.ComponentModel.DataErrorsChangedEventArgs>? ErrorsChanged;
+        System.Collections.IEnumerable GetErrors(string? propertyName);
```

Comment:
Below, `PropertyChanged` and `PropertyChanging` should be a `PropertyChangedEventHandler?`